packages/macro/src/macroJs.ts

```diff
@@ -352,8 +352,12 @@ export default class MacroJs {
           value = this.tokenizeTemplateLiteral(attrValue)
         } else if (this.types.isCallExpression(attrValue)) {
           value = this.tokenizeNode(attrValue)
+        } else if (this.types.isStringLiteral(attrValue)) {
+          value = attrValue.value
+        } else if (this.types.isExpression(attrValue)) {
```

Comment:
StringLiteral is also considered as Expression, so I moved StringLiteral check above. From ts,  after all the checks, the types fall through are limited to `babelTypes.RestElement | babelTypes.AssignmentPattern | babelTypes.ArrayPattern | babelTypes.ObjectPattern`,  not sure exactly how we want to handle them, so I kept the previous fallback case untouched.

Reply:
the `unknown` conversion is just to make ts happy, please let me know if you have other suggestions to handle the remaining cases.